src/taskgraph/run-task/run-task

```diff
@@ -1078,6 +1084,51 @@ def _display_python_version():
         b"Python version: "+ bytes(platform.python_version(), 'utf-8')
     )
 
+def _display_python_version_in_sub_process(subprocess: subprocess):
+    print_line(
+        b"setup",
+        b"pyhton --version: ",
+    )
```

Comment:
Nit: I'd recommend storing the command in a variable and then pretty print it here with:

```python
command = [python_executable, "--version",  "--version"]
print_line(b"setup", " ".join(command))
```

This reduces the likelihood of a typo and that prevents modifying the command at multiple locations the day we want to modify its flags, for instance.